app/src/main/java/org/p2p/wallet/debug/settings/DebugSettingsFragment.kt

```diff
@@ -92,6 +92,14 @@ class DebugSettingsFragment :
         }
     }
 
+    private fun onSettingsSwitchClicked(@StringRes titleResId: Int, isSelected: Boolean) {
```

Comment:
i would stick with the isChecked

Reply:
that's what I talked about  x D 20 minutes